src/styles/style-helpers.js

```diff
@@ -61,3 +61,35 @@ export const disableVisually = () => `
   opacity: 0.5;
   pointer-events: none;
 `;
+
+const add = (a, b) => a + b;
+const subtract = (a, b) => a - b;
+const multiply = (a, b) => a * b;
+const divide = (a, b) => a / b;
+
+const transformUnit = (values, transform, allowMultipleUnits = true) => {
```

Comment:
While I really like this function and what you did here, I think this is overkill. If the browser needs to recompute a bunch of styles over and over again, and for something as simple as `calc(2 *${theme.spacings.kilo}` we need to apply regexes and do string conversion multiple times, I would be afraid this is a waste of computational power.

Now I might totally be prematurely optimizing here, given the ridiculous amount of JS in our styles, but this feels like its taking JS too far down to the metal, if you know what I mean.

Do we really, really need this?